op-supervisor/supervisor/backend/cross/safe_start.go

```diff
@@ -15,100 +15,32 @@ type SafeStartDeps interface {
 	CrossDerivedToSource(chainID eth.ChainID, derived eth.BlockID) (source types.BlockSeal, err error)
 
 	DependencySet() depset.DependencySet
+
+	OpenBlock(chainID eth.ChainID, blockNum uint64) (ref eth.BlockRef, logCount uint32, execMsgs map[uint32]*types.ExecutingMessage, err error)
 }
 
 // CrossSafeHazards checks if the given messages all exist and pass invariants.
 // It returns a hazard-set: if any intra-block messaging happened,
 // these hazard blocks have to be verified.
-func CrossSafeHazards(d SafeStartDeps, chainID eth.ChainID, inL1Source eth.BlockID,
-	candidate types.BlockSeal, execMsgs []*types.ExecutingMessage) (hazards map[types.ChainIndex]types.BlockSeal, err error) {
-
-	hazards = make(map[types.ChainIndex]types.BlockSeal)
-
-	// Warning for future: If we have sub-second distinct blocks (different block number),
-	// we need to increase precision on the above timestamp invariant.
-	// Otherwise a local block can depend on a future local block of the same chain,
-	// simply by pulling in a block of another chain,
-	// which then depends on a block of the original chain,
-	// all with the same timestamp, without message cycles.
+func CrossSafeHazards(d SafeStartDeps, logger log.Logger, chainID eth.ChainID, inL1Source eth.BlockID,
+	candidate types.BlockSeal) (*HazardSet, error) {
```

Comment:
nit: I think I'm the only person around who cares about this (so feel free to totally ignore me), but I prefer when signatures follow one of:
- Whole signature on one line
- One argument per line
- arguments logically grouped per line (like pairs of arguments where applicable)
- arguments packaged into a struct (when there are a *lot* of related arguments and similar signatures in multiple places)

When linting complains about line-length, it is because the code may not be readable. While arbitrary line breaks satisfy the linter, I'm not sure they satisfy the purpose of the lint.